openwisp_utils/admin_theme/static/admin/js/menu.js

```diff
@@ -207,3 +208,15 @@ function initToolTipHandlers() {
     });
   });
 }
+
+function showActiveItems(){
+  var activeLink = document.querySelector(`.nav a[href="${window.location.pathname}"]`);
```

Comment:
Okay, I think this is why that `/` above was required.
From my point of view, why do we even need it?
Can't we add the active class to a button on click and remove the active class from all others. A simpler logic!
Do we really want highlighting when the users refreshes the page?

Reply:
I think we have to highlight group or item even on page refresh. @nemesisdesign What do you think?

Reply:
the page always refreshes when a link is clicked guys... it's not a single page app

Reply:
True 😂

Reply:
Ah, I am getting spoiled by frameworks! 😄